applications/OptimizationApplication/tests/algorithm_tests/analysis_based_tests/algorithm_gradient_projection/optimization_parameters.json

```diff
@@ -0,0 +1,146 @@
+{
+    "problem_data": {
+        "parallel_type": "OpenMP",
+        "echo_level": 0
+    },
+    "model_parts": [
+        {
+            "settings": {
+                "model_part_name": "Structure",
+                "domain_size": 3,
+                "input_filename": "Structure"
+            }
+        }
+    ],
+    "analyses": [
+        {
+            "name": "Structure_static",
+            "type": "kratos_analysis_execution_policy",
+            "settings": {
+                "model_part_names": [
+                    "Structure"
+                ],
+                "analysis_module": "KratosMultiphysics.StructuralMechanicsApplication",
+                "analysis_type": "StructuralMechanicsAnalysis",
+                "analysis_settings": {
+                    "@include_json": "primal_parameters.json"
+                },
+                "analysis_output_settings": {
+                    "nodal_solution_step_data_variables": [
+                        "DISPLACEMENT"
+                    ],
+                    "nodal_data_value_variables": [],
+                    "element_data_value_variables": [],
+                    "condition_data_value_variables": []
+                }
+            }
+        }
+    ],
+    "responses": [
+        {
+            "name": "mass",
+            "type": "mass_response_function",
+            "settings": {
+                "evaluated_model_part_names": [
+                    "Structure"
+                ]
+            }
+        },
+        {
+            "name": "strain_energy",
+            "type": "linear_strain_energy_response_function",
+            "settings": {
+                "evaluated_model_part_names": [
+                    "Structure"
+                ],
+                "primal_analysis_name": "Structure_static",
+                "perturbation_size": 1e-8
+            }
+        }
+    ],
+    "controls": [
+        {
+            "name": "thickness_control",
```

Comment:
the model is solid model and thickness control should not be used.

Reply:
True, I have made a new example with shell elements. 

![image](https://github.com/KratosMultiphysics/Kratos/assets/18362888/14928be9-3bd8-4cb7-9d13-dab81afeea13)